fairlearn/metrics/_metric_frame.py

```diff
@@ -814,7 +814,7 @@ def difference_ci(
         return value
 
     def ratio(
-        self, method: str = "between_groups", errors: str = "coerce"
+        self, method: Literal["between_groups", "to_overall"] = "between_groups", errors: Literal["raise", "coerce"] = "coerce"
```

Comment:
these lines are too long now. I think `black` should be handling these right.

Reply:
My bad, I haven't initialized the pre-commit hooks :eyes: